pallets/loans/src/entities/pricing/external.rs

```diff
@@ -94,6 +94,9 @@ pub struct ExternalActivePricing<T: Config> {
 
 	/// Current interest rate
 	pub interest: ActiveInterestRate<T>,
+
+	/// Related pool id
+	pool_id: T::PoolId,
```

Comment:
Not sure about adding this.

Pros:
- Easy to reason about the code: the loan has enough info to know itself the price.
- Do not populate a lot of calls everywhere by passing the `pool_id` parameter.

Cons:
- Requires a migration if it's done after nuking the storage.
- Extra stored bytes for that `pool_id` (minimum).

Reply:
Finally, I've removed it (here is the diff: [d0c6348](https://github.com/centrifuge/centrifuge-chain/pull/1474/commits/d0c6348f946ac4f08682c0c40712b5ad51cbfd9c)). But still doesn't know 100% the best choice here.